pkg/server/admin.go

```diff
@@ -554,7 +554,21 @@ func (s *adminServer) TableStats(
 
 	// Construct TableStatsResponse by sending an RPC to every node involved.
 	tableStatResponse := serverpb.TableStatsResponse{
-		NodeCount:  int64(len(nodeIDs)),
+		NodeCount: int64(len(nodeIDs)),
+		// TODO(mrtracy): The "RangeCount" returned by TableStats is more
```

Comment:
Well, it should be documented somehow. We have plenty of TODOs that are not unambiguously things we should do right now. I would add the action item.